docs/LV_client/sensors/mono_fisheye_camera_poly1.md

```diff
@@ -24,7 +24,7 @@ the sensor configuration.
 height x 4
  
 
-- **Camera configuration:**  Settings to configure a RGB camera
+- **Camera configuration:**  Settings to configure the camera sensor
  
 
 - **error out (Error Cluster):** Accepts error information wired from previously called VIs. This information can be used to decide if any functionality should be bypassed in the event of errors from other VIs. 
```

Comment:
It looks like its been fixed, but just in case. There's a "er VIs." at the bottom of the readthedocs build for this page. 
![Screen Shot 2020-11-02 at 10 15 35 AM](https://user-images.githubusercontent.com/41026694/97891557-8a21c900-1cf4-11eb-992a-d62b65eb9a98.png)

Reply:
Thanks, yes. This was fixed.